python/BUILD

```diff
@@ -49,6 +52,31 @@ py_native_lib_rename(
     visibility = ["//visibility:public"]
 )
 
+# Documentation generation
+py_binary(
+    name = "sphinx_runner",
+    srcs = [
+        "tool/sphinx/sphinx_html_builder_runner.py",
+    ],
+    main = "tool/sphinx/sphinx_html_builder_runner.py",
+    deps = [docs_requirement("sphinx")],
+    visibility = ["//visibility:public"]
+)
+
+sphinx_docs(
+    name = "driver_docs",
+    script = ":sphinx_runner",
+    target = ":assemble-pip39",
+    srcs = glob(["tool/sphinx/source/*"]),
+    out = "docs/build",
+)
+
+html_docs_parser(
+    name = "driver_adoc",
+    data = ":driver_docs",
+    language = "python",
+)
+
```

Comment:
For hermeticity we run `sphinx` as a python module instead of the command line tool:
- `sphinx_runner`: a binary that runs `sphinx`
- `driver_docs`: generates HTML docs
- `driver_adoc`: produces AsciiDoc files from HTML docs and put them into the docs directory